modules/localstorage.js

```diff
@@ -0,0 +1,9 @@
+function getStoredBooks() {
```

Comment:
- Use an arrow function everywhere in this project. Please make the necessary changes